packages/extension-mention/src/mention.ts

```diff
@@ -9,6 +9,7 @@ export type MentionOptions = {
     options: MentionOptions,
     node: ProseMirrorNode,
   }) => string,
+  deleteOnBackspace: boolean,
```

Comment:
I guess we would need to add this new option to the documentation then? Otherwise looks good to me.